src/sigal/gallery.py

```diff
@@ -935,8 +936,23 @@ def remove_files(self, medias):
 
     def process_dir(self, album, force=False):
         """Process a list of images in a directory."""
+        def forcing(a):
+            if force is None:
+                return False
```

Comment:
This case should never happen.

Reply:
Yup, I was trying to just cover any cli cases.